drone-webdriver/src/test/java/org/jboss/arquillian/drone/webdriver/binary/downloading/source/GitHubSourceLatestReleaseFromTagsTestCase.java

```diff
@@ -30,6 +32,8 @@
 import static org.mockito.Mockito.times;
 import static org.mockito.Mockito.verify;
 
+@Ignore("For the evaluation the PhantomJSGitHubBitbucketSource is used. But it doesn't use this logic any more."
```

Comment:
For gecko there is: https://github.com/arquillian/arquillian-extension-drone/blob/master/drone-webdriver/src/test/java/org/jboss/arquillian/drone/webdriver/binary/downloading/source/GitHubSourceLatestReleaseTestCase.java